101-dashing-Ecommerce/styles/hero.css

```diff
@@ -0,0 +1,25 @@
+/* hero.css */
+.hero {
+    background: url('hero-bg.jpg') no-repeat center center/cover;
+    color: black;
+    padding: 100px 0;
+    text-align: center;
+}
+
+.hero h1 {
+    font-size: 48px;
+    margin-bottom: 20px;
+}
+
+.hero p {
+    font-size: 24px;
+    margin-bottom: 40px;
+}
+
+.hero .btn {
+    padding: 10px 20px;
+    background: #ff6f61;
```

Comment:
Instead of using hard-coded values in CSS, I recommend using CSS variables for easier maintenance. Learn more here: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties